index.html

```diff
@@ -0,0 +1,241 @@
+                <p class="bottom-text">Gate lectures by Ravindraba...<br />Recommended for you</p>
+               </div>
+           </div>
+           <hr />
+
+           <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/img3.PNG" alt="img3"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text">Contributing to Open Source <br />Part II: The real way</p>
+             <p class="bottom-text">The Odin Project<br />32K views</p>
+
+            </div>
+
+        </div>
+
+
+        <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/img4.PNG" alt="img4"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text">Mastery: How To Learn Anything Fast</p>
+             <p class="bottom-text">Ideas and inspiration<br />Recommended for you</p>
+            </div>
+
+        </div>
+
+        <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/img5.PNG" alt="img5"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text">Microsoft word tutorial|How to insert images into word...</p>
+             <p class="bottom-text">ramat maulana<br />Recommended for you</p>
+
+            </div>
+
+        </div>
+
+        <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/img6.PNG" alt="img6"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text">Greatest Liverpool goals of the decade| Gerard, Suarez...</p>
+             <p class="bottom-text">Liverpool FC<br />1M views</p>
+
+            </div>
+
+        </div>
+
+        <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/qazi.PNG" alt="qazi"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text">How to Land Freelance Jobs with Python (even if you're a beginner)</p>
+             <p class="bottom-text">Clever Programmer<br />12K views</p>
+
+            </div>
+
+        </div>
+
+        <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/threeM.PNG" alt="threeM"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text" >Can You Become a Programmer in Only Three Months</p>
+             <p class="bottom-text">Andy Sterkowitz<br />200K</p>
+
+            </div>
+
+        </div>
+
+        <div class="inner-grid">
+            <div class="thumbnail">
+                <img src="images/top.PNG" alt="top"/>
+            </div>
+
+            <div class="description">
+             <p class="top-text">Top Programming Languages in 2020</p>
+             <p class="bottom-text">Liverpool FC<br />156K views</p>
+
+            </div>
+
+        </div>
+
+       </div>
+
+
+    </div>
+
+    </body>
```

Comment:
Since this project's goal is to learn how to use GitHub features for reviews and how to communicate in a professional way, I think you can make the following changes:

- [ ] It is required that you use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments. This means that you should the comment under a line or block of code that fits the context the comment is about. Check for example of an inline comment below.

- [ ] It is also important that you phrase the comments politely, as suggestions instead of orders.
For example:
This is a polite suggestion: **"I think it would be better to use semantic tags"**
This reads like an order: **"Use semantic tags"**

- [ ] Also, you should always include an explanation, a "why", for each requested change. 
For example:
This is a well-explained request: **_Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible_**
This request lacks a "why": **_Why don't you use an article tag here instead of a div?_**

- [ ] Lastly, your comments must be clear and easy to understand. 
For Example:
This is a clear comment: **_The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?_**
This is a vague comment: **_The README is not descriptive, why don't you make it better?_**